app/src/main/java/de/westnordost/streetcomplete/quests/address/AddHousenumber.kt

```diff
@@ -129,6 +130,10 @@ class AddHousenumber : OsmElementQuestType<HousenumberAnswer> {
     override fun isApplicableTo(element: Element): Boolean? =
         if (!buildingsWithMissingAddressFilter.matches(element)) false else null
 
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes, ways, relations with (addr:housenumber or addr:housename) " +
+            "and !amenity and !craft and !leisure and !office and !shop and !tourism")
```

Comment:
Please use multiline strings to stay consistent with other element filters and to avoid string concatenation.


```suggestion
        getMapData().filter("""
            nodes, ways, relations with
              (addr:housenumber or addr:housename)
              and !amenity and !craft and !leisure and !office and !shop and !tourism
        """)
```